docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+services:
+  nginx:
+    image: "nginx:alpine"
+    ports:
+      - 5043:443
+    restart: unless-stopped
+    links:
+      - registry:registry
+    volumes:
+      - ./auth:/etc/nginx/conf.d
```

Comment:
@manekenpix where should we put our files in the host?  It might make sense to have `./nginx` but I don't think `auth` makes as much sense for these.

Reply:
Maybe somewhere in `/etc`, like `/etc/letsencrypt`?

Reply:
I think a `config` folder would be ok?

Reply:
I had thought the same thing originally, but after going back and looking at the documentation I saw this. 
![image](https://user-images.githubusercontent.com/59970326/156011440-ff596c7b-a09b-43b3-8655-8e2cf1e754ad.png)
from here https://docs.docker.com/registry/recipes/nginx/#setting-things-up

So I decided to go with the current folder structure. I can create and move it back to a config folder though?